src/common/ini_handler.c

```diff
@@ -18,44 +19,72 @@ static int ini_handler_func(void *user, const char *section, const char *name, c
 
     ETH_config_t *tmp_config = (ETH_config_t *)user;
 
-    if (ini_string_match(section, "lan_ip4", name, "type")) {
-        if (strncasecmp(value, "DHCP", 4) == 0) {
+    if (ini_string_match(section, "eth::ipv4", name, "type")) {
+        if (strncmp(value, "DHCP", 4) == 0 || strncmp(value, "dhcp", 4) == 0) {
```

Comment:
`dhcp` does not correspond to description of task BFW-1939 - beware upper/lower case, this has always been a major source of confusion while using these INI files. I'd recommend doing `stricmp` to making the configuration less pain/error prone

The other keywords here are dtto.

Reply:
understand. From discussion with @ondratu we keep such "constants" upper case

Reply:
BFW-1939 will be updated accordingly?
What will be the backwards compatibility with existing ini files?

Reply:
example ini files were always with upper case. if to keep complete backward compatibility both upper and lower case must be kept